extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -724,8 +724,11 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 					vscode.window.showInformationMessage(localize("sql.migration.starting.migration.message", 'Starting migration for database {0} to {1} - {2}', this._migrationDbs[i], this._targetServerInstance.name, this._targetDatabaseNames[i]));
 				}
 			} catch (e) {
+				vscode.window.showErrorMessage(
+					localize('sql.migration.starting.migration.error', "Error message: '{0}'.  stack:'{1}'",
```

Comment:
Generally showing the user a stack trace isn't very useful since they can't do anything about it. I would suggest instead just indicating that something went wrong and including the error message. You can print the error to the console if you think that's useful for debugging purposes.

Reply:
Updated to:
![image](https://user-images.githubusercontent.com/61598682/120699498-4d25dc80-c465-11eb-9716-55c8fd0927fe.png)